main.js

```diff
@@ -222,14 +275,20 @@ function createElement(tagName, attributes, children) {
 
 function renderApp(state) {
   var $view = document.querySelector('[data-view="' + state.view + '"]')
+  var $cart = document.querySelector('[data-view="cart"]')
+  $cart.innerHTML = ''
   $view.innerHTML = ''
-  if (state.view === 'details') {
-
+  if (state.view === 'cart') {
     $view.appendChild(renderCartCount(state.cart))
+    $view.appendChild(renderCartPage(state.cart))
```

Comment:
If you are always doing this `$view.appendChild(renderCartPage(state.cart))` you can move it outside of the `if...else`